app/models/miq_server.rb

```diff
@@ -487,11 +487,24 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
-  def self.audit_managed_resources
-    total_vms   = VmOrTemplate.active.count
-    total_hosts = Host.active.count
+  def self.managed_resources
+    {
+      :vms                     => VmOrTemplate.active.count,
```

Comment:
Suggest this be changed to `Vm.active.count` to more accurately reflect reality.